relationships/synthesis/EXT-SERVICE-to-INFRA-ELASTICACHE-REDIS-CLUSTER

```diff
@@ -0,0 +1,54 @@
+relationships:
+  - name: extServiceCallsInfraElastiCacheRedisCluster
+    version: "1"
+    origins:
+      - OpenTelemetry
+    conditions:
```

Comment:
Looks like we have data matching this condition: https://staging.onenr.io/0ZQWbOP1pjW

Reply:
Attributes are also present in the `Span` events: https://staging.onenr.io/0nQxLveXdQV
Is there any reason to use `Metric` over `Span`? Could we use both?